infra.bs

```diff
@@ -1042,6 +1042,19 @@ as 200/`<code>OK</code>`.
   <p class=note>The conventions used in this step are those of the JavaScript specification.
 </ol>
 
+<p>To <dfn export>JSON stringify and UTF-8 encode to bytes</dfn> a given JavaScript value
+<var>value</var>, run these steps:
+
+<ol>
+ <li>
+  <p>Let <var>jsonString</var> be the result of
+  <a abstract-op>Call</a>(<a>%JSONStringify%</a>, undefined, « <var>value</var> »).
+  [[!ECMA-262]]
+  <p class=note>The conventions used in this step are those of the JavaScript specification.
```

Comment:
Maybe add `<p class=note>Since no additional arguments are passed to <a>%JSONStringify%</a>, the resulting string will have no whitespace inserted</p>`

Reply:
I added your suggested Note to the existing Note. if not OK will make it a separate note, see 7d94fda.